### DISCUSSION TITLE
Information sharing

### DISCUSSION THREAD

**Discussion by David-DL** [reportportal/reportportal#1553] [Q&A] [answered]
As far as I've seen in the docs and all the information, looks like everything ReportPortal does is executed locally, on-premises. But just wanted to make sure by asking here. Isn't there any data shared to you about the executions for model training and any other purposes?
I ask this mainly from the confidential data handling point of view, as my company is considering implementing it.

Thanks!

**Comment by evjlobanova** [comment, marked as answer]
Hello, all data is stored on your servers. 
No data is shared with anybody.
We are collecting information about events via Google Analytics, but it can be switched off
